projects/core/styles/mixins/mixins.less

```diff
@@ -428,7 +428,9 @@
 .scroll-behavior(@mode: smooth) {
     scroll-behavior: @mode;
 
-    :host-context(tui-root._reduced-motion) & {
+    &._reduced-motion,
+    ._reduced-motion > &,
+    :host-context(._reduced-motion) & {
```

Comment:
I don't like this selector, it's better not to reach outside `tui-root`. Let's roll this back and just keep smooth-scrolling always on, it's in the demo app anyway, not in the addon-doc.

Reply:
fixed